_includes/head.html

```diff
@@ -1,6 +1,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:description" content="UnetStack3 is a major milestone in the evolution of UnetStack, debuting a software-defined modem that can run on your laptop or on an embedded computer" />
+    <meta name="twitter:title" content="Wnat's new in UnetStack3" />
```

Comment:
Typo

Reply:
Modified the title to be fetched from the post.title.